Lib/multiprocessing/queues.py

```diff
@@ -98,6 +122,9 @@ def put(self, obj, block=True, timeout=None):
     def get(self, block=True, timeout=None):
         if self._closed:
             raise ValueError(f"Queue {self!r} is closed")
+        if self._is_shutdown_immediate() or\
+            (self._is_shutdown() and self.empty()):
```

Comment:
```suggestion
        if (
            self._is_shutdown_immediate()
            or (self._is_shutdown() and self.empty())
        ):
```

See [PEP 8](https://peps.python.org/pep-0008/#maximum-line-length): backslash line-breaks are discouraged where possible